### PR TITLE
add pre-requirement

### PR DESCRIPTION
**Pull request by zhangcoldwater**
require golang version 1.5 or later

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Read the [Roadmap](./ROADMAP.md).
 
 - __Pre-requirement__  
 ```
+golang version 1.5 or later
```
Comment (zimulala) [inline README.md]: You should replace 1.5 to 1.42.

Comment (siddontang) [inline README.md]: @zhangcoldwater 
code block should contain shell command,  not a description sentence.  
maybe you can put it out.

**Comment by zhangcoldwater** [comment]
make is ok with go1.4 or later, but make interpreter failed with go1.4.2

**Comment by zimulala** [comment]
s/golang/Go/g
And add . in the end of the sentence

File: README.md [hunk 2]
```diff
@@ -36,6 +36,8 @@ Read the [Roadmap](./ROADMAP.md).
 ## Quick start
 
 - __Pre-requirement__  
+
+golang version 1.5 or later
```
Comment (siddontang) [inline README.md]: Go environment. Currently a 64-bit version of go >= 1.5 is required.

seems better.

Comment (zhangcoldwater) [inline README.md]: ok

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**